test/auto/injectorSpec.js

```diff
@@ -237,6 +237,15 @@ describe('injector', function() {
     });
 
 
+    // Only Chrome and Firefox support this syntax.
+    if (/chrome|firefox/i.test(navigator.userAgent)) {
+      it('should be possible to annotate functions that are declared using ES6 syntax', function() {
+        // The function is generated using `eval` as just having the ES6 syntax can break some browsers.  
+        expect(annotate(eval('({ fn(x) { return; } })').fn)).toEqual(['x']);
```

Comment:
Without the `eval` trick, Safari 8.0.7 (10600.7.12)  [This is the latest Safari version]
throws this error when parsing the file to run the tests
```
SyntaxError: Unexpected token '('. Expected a ':' following the property name 'fn'.
```
After throwing this error, no tests are executed.